### ISSUE TITLE
[Request] search option in UI for warp travel

### ISSUE THREAD

**Issue by thestargateman** [zmaster587/AdvancedRocketry#1815]
in worlds with a lot of custom planets added it can be difficult to find them, the holographic planet selector is useful but i play on a world with 100+ systems and looking through all of them can be a pain. A simple search option for planets / systems would be beneficial.

**Comment by Gliese-832-c** [comment]
Yes, this should be added. The planet/star list (bottommost button in the top left part of the screen) is broken, it can at most display 6 or 7 planets/stars, and you can't scroll. While I don't have 100+ systems, I do have quite a few at 21, with some of them, currently two, having more than 7 planets. A search option would be appreciated.
